andela/indiana-ah-backend
# #163426524 Implement social authentication

#### What does this PR do?
Implements social authentication for users

#### Description of Task to be completed?
add Google authentication 
add Facebook authentication
add Twitter authentication

#### How should this be manually tested?
Clone the repo, cd into it, run npm install to install all dependencies. Check the env.sample file and input the required api keys and secret. Run `npm run dev` to start the server on port 3000. Navigate to any of the following routes below to sign up or log in with your social account.

- GET /auth/google
- GET /auth/facebook
- GET /auth/twitter

**With test suite:**
- Run `npm test`

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
#163426524

#### Screenshots
<img width="490" alt="screen shot 2019-02-11 at 2 50 54 pm" src="https://user-images.githubusercontent.com/31392460/52569679-09c97f00-2e12-11e9-985d-7c075444d868.png">
<img width="1429" alt="screen shot 2019-02-11 at 2 51 55 pm" src="https://user-images.githubusercontent.com/31392460/52569696-164dd780-2e12-11e9-9df6-59c169248353.png">
